src_assets/windows/assets/shaders/directx/include/base_vs.hlsl

```diff
@@ -25,7 +25,7 @@ vertex_t generate_fullscreen_triangle_vertex(uint vertex_id, float2 subsample_of
         float2 rotation_center = { 0.5, 0.5 };
         tex_coord = round(rotation_center + mul(rotation_matrix, tex_coord - rotation_center));
 

```

Comment:
/*bitwise operator used as modulus operation on integer does not compile optimally and generates warnings, bitwise operation "& 1" is equivalent to "% 2" in a form that results in optimized lower level code*/